centinel/client.py

```diff
@@ -37,7 +37,7 @@ def get_input_file(self, experiment_name):
         return os.path.join(self.config['dirs']['data_dir'], input_file)
 
     def load_input_file(self, name):
-        input_file = self.get_input_file(name)
+        input_file = self.get_input_file("%s.txt" %(name))
```

Comment:
I don't think this is a good idea. We go from forcing it to have no extension to forcing it to end in .txt. If we also want to check .txt files, then we should update load_input_file to check both cases (with .txt and no extension), or ideally, check any extension. It's also unclear to me if this update belongs in load_input_file or get_input_file